src/index.css

```
@@ -250,6 +250,47 @@ body {
   font-family: var(--font-editor);
 }
 
+.comet-lexical-tree-view {
+  max-height: min(24rem, 45vh);
+  overflow: auto;
+  margin: 0;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)